src/logical.js

```diff
@@ -28,4 +28,9 @@ export const greaterOrEqual =
 export const lesser = (inputA, inputB) => binary(inputA, inputB, (a, b) => (a < b ? 1 : 0));
 export const lesserOrEqual =
     (inputA, inputB) => binary(inputA, inputB, (a, b) => (a <= b ? 1 : 0));
-export const not = (input) => logicalNot(input);
+export const logicalAnd =
+    (inputA, inputB) => binary(inputA, inputB, (a, b) => (!!a && !!b ? 1 : 0));
+export const logicalNot = (input) => logicalNotImpl(input);
+export const logicalOr =
+    (inputA, inputB) => binary(inputA, inputB, (a, b) => (!!a || !!b ? 1 : 0));
+export const logicalXor = (inputA, inputB) => binary(inputA, inputB, (a, b) => (!!a ^ !!b));
```

Comment:
Why is this one not like others?
```suggestion
export const logicalXor = (inputA, inputB) => binary(inputA, inputB, (a, b) => (!!a ^ !!b ? 1 : 0));
```

Reply:
I missed `^` is a bitwise operator.